app/assets/stylesheets/forms.scss

```diff
@@ -67,3 +67,14 @@
 .vertical-drag {
   cursor: ns-resize;
 }
+
+.inset-text-field {
+  border-left: 5px solid #bcbcbc;
+  padding: 6px 0 6px 10px;
+  position: relative;
+  margin-left: 25px;
+
+  .non-bold-label {
```

Comment:
I'd suggest naming this to `.inset-text-field-label` as it seems a bit surprising that this class would only work in a nesting situation (and the naming is perhaps too specific about what the style does rather than what the element is)

Reply:
Ah! Ok, I tend to read nested classes in context. So this would be `.inset-text-field-non-text-field` - in my head. Happy to change it.  Fixed up.